documentation/book/con-custom-resources-example.adoc

```diff
@@ -64,16 +64,19 @@ kind: KafkaTopic <1>
 metadata:
   name: my-topic
   labels:
-    strimzi.io/cluster: my-cluster
-spec: <2>
+    strimzi.io/cluster: my-cluster <2>
+spec: <3>
   partitions: 1
   replicas: 1
   config:
     retention.ms: 7200000
     segment.bytes: 1073741824
 ----
 <1> The `kind` and `apiVersion` identify the CRD of which the custom resource is an instance.
-<2> The spec shows the number of partitions and replicas for the topic as well as configuration for the retention period for a message to remain in the topic and the segment file size for the log.
+<2> A label, applicable only to `KafkaTopic` and `KafkaUser` resources, that defines the name of the Kafka cluster (derived from the name of the `Kafka` resource) to which a topic or user belongs.
```

Comment:
"Derived from the name of the `Kafka` resource" is a bit ambiguous. Does this mean that the label is the same as the name of the Kafka cluster?

Reply:
Yes it is.

Reply:
A label, applicable only to KafkaTopic and KafkaUser resources, that defines the name of the Kafka cluster (which is the same as the name of the Kafka resource) to which a topic or user belongs.